src/ts/background/services/keyring-vault.ts

```diff
@@ -15,6 +15,7 @@ class KeyringVault {
   private _keyring?: KeyringInstance
   private _password?: string
   private _mnemonic?: string
+  private accountsToInject?: Accounts
```

Comment:
Minor: I would probably give a more generic name e.g accounts

Reply:
If we change Accounts to SimpleAccounts, then we change rename accountsToInject to simpleAccounts to be distinguished from normal "accounts"